frontend/test/ui/pages/desktop/home.py

```diff
@@ -0,0 +1,21 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Home(Base):
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    class Header(Region):
+
+        _root_locator = (By.CLASS_NAME, 'banner')
+
+        _copter_locator = (By.CLASS_NAME, 'copter')
```

Comment:
That's fine, but we can just use the one in `Base`, and `Home` can just be a simple subclass.